PhotoPlusFrontend/PhotoPlus/src/app/components/products/products.component.scss

```diff
@@ -77,3 +77,32 @@ img {
 :focus {
     outline: 0px;
 }
+
+.dropdown-option {
+  margin: 10px;
+  color: black;
+}
+
+.select.mat-selected {
+  color: white !important;
+}
+
+::ng-deep .mat-select-value-text {
+  color: lightslategrey !important;
+}
+
+::ng-deep .mat-select-arrow {
+  color: lightslategrey;
+}
+
+::ng-deep .mat-form-field.mat-focused .mat-form-field-ripple  {
+  background-color: lightslategrey !important;
+}
+
+::ng-deep .mat-form-field.mat-focused .mat-select-arrow {
+  color: lightslategrey !important;
+}
```

Comment:
I;ve removed all ::ng-deeps some time ago cause of their depreciation, why u need to use them?

Reply:
It's the only way to change mat-select styles. Angular team didn't provide any replacement for  ::ng-deep (instead of ViewEncapsulation which doesn't work in this case) and will keep ::ng-deep available for long time.

Reply:
Dude never again do u repeat' that 'is impossible" stackOverflow nonsense & look up how i bypassed these encapsulations myself